tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +346,195 @@ def test_del_obj_tag_40436(self):
+
+        LOGGER.info("Step 3: Perform PUT Object Tagging for %s "
+                    "with a tag key-value pair with Version ID %s",
+                    self.object_name, latest_v)
+        resp = s3_ver_tlib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_tag=self.ver_tag,
+                                              version_id=latest_v,
+                                              versions_dict=self.versions)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 3: Performed PUT Object Tagging for %s with a tag key-value pair",
+                    self.object_name)
+
+        LOGGER.info("Step 4: Perform GET Object Tagging for %s with versionId=%s",
+                    self.object_name, latest_v)
+        put_tag = self.ver_tag[self.object_name][latest_v][-1]
+        resp = s3_ver_tlib.delete_object_tagging(bucket_name=self.bucket_name,
+                                                 s3_ver_test_obj=self.s3_ver_obj,
+                                                 s3_tag_test_obj=self.s3_tag_obj,
+                                                 object_name=self.object_name,
+                                                 version_id=latest_v)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1][0]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+        LOGGER.info("Step 4: Performed GET Object Tagging for %s with versionId=%s is %s",
+                    self.object_name, latest_v, get_tag)
+
+        non_existing_version = ''.join(random.sample(latest_v, len(latest_v)))
+        LOGGER.info("Step 5: Perform DELETE Object Tagging for object1  %s "
+                    "with a tag key-value pair for non-existing "
+                    "versionId=%s",
+                    self.object_name, non_existing_version)
+        resp = s3_ver_tlib.delete_object_tagging(bucket_name=self.bucket_name,
+                                                 s3_ver_test_obj=self.s3_ver_obj,
+                                                 s3_tag_test_obj=self.s3_tag_obj,
+                                                 object_name=self.object_name,
+                                                 version_id=non_existing_version)
+        # For non-existing version ID , expecting "404 Not Found (NoSuchKey)"
+        assert_utils.assert_false(resp[0], resp)
+        assert_utils.assert_in(NO_SUCH_KEY_ERR, resp[1].message)
+        LOGGER.info("Step 5: Performed DELETE Object Tagging for %s with versionId=%s",
+                    self.object_name, latest_v)
```

Comment:
not latest_v but non_existing_version. 

Also please mention in step for ...sith non-existing versionId=%s...